Packs/ShiftManagement/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,6 @@
+## [Unreleased]
+- Add the GetShiftsPerUser automation
+
+### Scripts
+- __GetOnCallHoursPerUser__
+%%UPDATE_RN%%
```

Comment:
```suggestion
- Add the GetShiftsPerUser automation
```

Reply:
@agamm You committed this too?

Reply:
https://github.com/demisto/content/blob/471e6f9cb4e4b2ad5f0df555d292dfc1f8567d5c/Packs/ShiftManagement/ReleaseNotes/1_1_0.md here is the latest file.